Mage.Client/src/main/resources/card-pictures-tok.txt

```diff
@@ -1809,4 +1810,32 @@
 |Generate|TOK:DDU|Elf Warrior|||ElfWarriorToken|
 |Generate|TOK:DDU|Myr|||MyrToken|
 |Generate|TOK:DDU|Thopter|1||ThopterColorlessToken|
-|Generate|TOK:DDU|Thopter|2||ThopterColorlessToken|
\ No newline at end of file
+|Generate|TOK:DDU|Thopter|2||ThopterColorlessToken|
+
+# IMA
+|Generate|TOK:IMA|Angel|||AngelToken|
+|Generate|TOK:IMA|Beast|||BeastToken2|
+|Generate|TOK:IMA|Bird|||BirdToken|
+|Generate|TOK:IMA|Djinn Monk|||DjinnMonkToken|
+|Generate|TOK:IMA|Dragon|1||DragonEggDragonToken|
+|Generate|TOK:IMA|Dragon|2||DragonToken2|
+|Generate|TOK:IMA|Spirit|||SpiritWhiteToken|
+
+# CM2
+|Generate|TOK:CM2|Bird|||BirdToken|
+|Generate|TOK:CM2|Elemental Shaman|||ElementalShamanToken|
+|Generate|TOK:CM2|Phyrexian Germ|||PhyrexianGermToken|
+|Generate|TOK:CM2|Goat|||GoatToken|
+|Generate|TOK:CM2|Goblin|||GoblinToken|
+|Generate|TOK:CM2|Knight|||HuntedDragonKnightToken|
+|Generate|TOK:CM2|Lightning Rager|||LightningRagerToken|
+|Generate|TOK:CM2|Myr|||MyrToken|
+|Generate|TOK:CM2|Pentavite|||PentaviteToken|
+|Generate|TOK:CM2|Saproling|||SaprolingToken|
+|Generate|TOK:CM2|Shapeshifter|||CribSwapShapeshifterWhiteToken|
+|Generate|TOK:CM2|Spirit|||SpiritWhiteToken|
+|Generate|TOK:CM2|Triskelavite|||TriskelaviteToken|
+|Generate|TOK:CM2|Tuktuk the Returned|||TuktukTheReturnedToken|
+|Generate|TOK:CM2|Phyrexian Wurm|1||WurmWithDeathtouchToken|
+|Generate|TOK:CM2|Phyrexian Wurm|2||WurmWithLifelinkToken|
```

Comment:
`WurmWithLifelinkToken` must have `setTokenType(2)` code (all tokens with 2+ index numbers must have that code).

Reply:
WurmWithLifelinkToken already has setTokenType(2) in its code for CM2, as that was there before this PR. See below:
https://github.com/magefree/mage/blob/80f6ceccff24a0aebf17060c6974f6f18971b0fd/Mage/src/main/java/mage/game/permanent/token/WurmWithLifelinkToken.java#L38